pkg/api/handlers/types.go

```diff
@@ -261,17 +262,17 @@ func ImageDataToImageInspect(ctx context.Context, l *libpodImage.Image) (*ImageI
 		//	StdinOnce:       false,
 		Env: info.Config.Env,
 		Cmd: info.Config.Cmd,
-		//	Healthcheck:     nil,
+		//Healthcheck: l.ImageData.HealthCheck,
 		//	ArgsEscaped:     false,
 		//	Image:           "",
-		//	Volumes:         nil,
-		//	WorkingDir:      "",
-		//	Entrypoint:      nil,
+		Volumes:    info.Config.Volumes,
+		WorkingDir: info.Config.WorkingDir,
+		Entrypoint: info.Config.Entrypoint,
 		//	NetworkDisabled: false,
 		//	MacAddress:      "",
-		//	OnBuild:         nil,
-		Labels: info.Labels,
-		//	StopSignal:      "",
+		//OnBuild:    info.Config.OnBuild,
```

Comment:
Same - still commented out?

Reply:
I am trying to figure out how to get this to work.  I would prefer to get this merged and leave the comment for someone in the future working on the TODO.